sentences/fr/_common.yaml

```
@@ -668,6 +668,7 @@ expansion_rules:
   reprends: "(remets|remettre|reprends|reprend|reprendre|reprise|relance|relancer)"
   cree: "(créer|crée)"
   supprime: "(supprime|supprimer)"
+  annule: "(annule|annuler)"
```

```suggestion
  supprime: "(supprime|supprimer|annule|annuler)"
```

Let's bundle "annule" and "supprime" on the same expansion rule.
They have very close meaning.

Bonus: You will not need to add sentences if you do that, it will be taken care of by the expansion rule.